### DISCUSSION TITLE
Feedback on closed issues

### DISCUSSION THREAD

**Discussion by georgwittberger-telekom-mms** [telekom/scale#2121] [General]
How do you handle feedback on issues that you closed as "fixed" or "won't fix"? At the moment contributors do not have any option to re-open the issue in case they are not satisfied with the solution.

For example, some of my bug issues have been closed recently, claiming they have been fixed. In the end, it turned out that the issues still persist, so usually I would just re-open the existing issues. What should we do in this case?

- Option 1 (what I do at the moment): Just comment on the closed issue and explain why it is not solved yet.
- Option 2: Allow contributors to re-open issues. Not sure how this whould work regarding GitHub permissions because anyone could possibly change issue status.

If just commenting on the issue is the way to go, do you need any special kind of comment in case the issue must be re-opened? Should contributors mention the most recent assignee?

**Comment by acstll** [comment]
Option 1 should work, this is how we've been handling it until now. It can perfectly be we haven't been able to handle those because of resources. We understand this is not ideal.

A third option would be to open a new issue referencing the old one. This could be a bit annoying because things would be harder to track but it's an option nevertheless.

> If just commenting on the issue is the way to go, do you need any special kind of comment in case the issue must be re-opened?

yes, nothing special…

> Should contributors mention the most recent assignee?

you can mention anybody you think appropriate.

Thanks for the kind message and sorry for the super late reply. It's highly appreciated that you take the time to report those issues.

**Reply by georgwittberger-telekom-mms** [reply]
Thanks @acstll for the explanation. So we will stick with option 1 and just comment on existing issues.

I would not like to open separate issues for problems already reported. As you write, it is also for you annoying to have duplicate issues and makes tracking problems harder.
